tests/common.sh

```diff
@@ -219,6 +217,7 @@ function error {
   kubectl -n my-ripsaw logs --tail=40 -l name=benchmark-operator -c benchmark-operator
   echo "Ansible sidecar Logs"
   kubectl -n my-ripsaw logs -l name=benchmark-operator -c ansible
+  kubectl -n my-ripsaw delete benchmark --all
```

Comment:
We shouldn't have to do this here. It should be done when being cleaned up when it hits the finish function.